src/current/v24.3/ldap-authorization.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Configure LDAP Authorization
+summary: Learn how to configure role-based access control (authorization) using LDAP with directory services like Active Directory.
+toc: true
+---
+
+LDAP authorization allows CockroachDB to use group memberships accessed using LDAP, for example, from an Active Directory server, as the source of truth for [role](https://www.cockroachlabs.com/docs/stable/security-reference/authorization#roles) assignments. When enabled:
+
+1. During login, CockroachDB retrieves the user's group memberships using LDAP.
+2. Groups are mapped to CockroachDB roles using the group's Common Name (CN).
+3. The user is automatically granted membership to corresponding roles.
+4. Any existing role memberships not matched to LDAP groups are revoked.
+
+## Prerequisites
+
+- Enable [LDAP Authentication]({% link v24.3/ldap-authentication.md %}).
+
+## Configuration
+
+### Step 1: Enable LDAP Authorization
+
+Add the `ldapgrouplistfilter` parameter to your HBA configuration. The configuration includes two important LDAP filters:
+
+1. `ldapsearchfilter`: Determines which users can authenticate
+2. `ldapgrouplistfilter`: Defines which groups should be considered for authorization
+
+Here's a basic example:
+
+~~~ sql
+SET CLUSTER SETTING server.host_based_authentication.configuration = '
+host    all    all    all    ldap    ldapserver=ldap.example.com 
+    ldapport=636 
+    "ldapbasedn=ou=users,dc=example,dc=com" 
+    "ldapbinddn=cn=readonly,dc=example,dc=com" 
+    ldapbindpasswd=readonly_password 
+    ldapsearchattribute=uid 
+    "ldapsearchfilter=(memberof=cn=cockroachdb_users,ou=groups,dc=example,dc=com)"
+    "ldapgrouplistfilter=(objectClass=groupOfNames)"';
+~~~
+
+For more precise control, you can configure these filters to match your security requirements:
```

Comment:
I wonder if it is worth just having an H2 "Examples" heading near the end of the page, where you put these sections. This is what we have tended to do in other pages. It's a little weird to me to have all these examples and then have "Step 2" after them.